SQL/New_patches/2019-05-03-deprecate_useprojects.sql

```diff
@@ -6,10 +6,10 @@ SELECT 'loris'
 WHERE NOT EXISTS (SELECT * FROM Project);
 
 -- associate all subprojects to the loris project by default if project table was empty
-INSERT INTO project_rel (ProjectID,SubprojectID)
-SELECT ProjectID, SubprojectID 
-FROM Project JOIN subproject 
-WHERE Project.Name='loris';
+INSERT IGNORE INTO Project (ProjectID, Name)
```

Comment:
@PapillonMcGill you replaced the wrong query

Reply:
fixed!